data/json/items/tool_armor.json

```diff
@@ -786,6 +786,51 @@
       "target": "wearable_atomic_light"
     }
   },
+  {
+    "id": "armor_EOD",
+    "type": "TOOL_ARMOR",
+    "category": "armor",
+    "copy-from": "armor_EOD_light",
+    "name": "EOD suit",
+    "description": "This is a massive suit constructed from kevlar and nomex for explosive ordnance disposal, complete with an armored full face helmet and boots.  This garment will provide significant protection from overpressure, fragmentation, impact and heat.  The electronically shielded helmet contains a camera, a two-way radio, and a headlamp, all of which can be voice-activated for redundancy.  This likely has nothing to do with the suit's lack of gloves.",
+    "weight": "31298 g",
+    "volume": "10000 ml",
+    "price": 760000,
+    "covers": [ "HEAD", "MOUTH", "EYES", "TORSO", "ARMS", "LEGS", "FEET" ],
```

Comment:
Which suit is this based on? Every real bomb suit I know of has detachable boots and helmet. Please avoid making "covers everything" single pieces unless there's a strong reason to, it's both less realistic and screws over mutated characters.

Reply:
Split up the items into most of their individual parts. They're based on the Med-Eng EOD 10 and TAC6E.